src/connectors/radiojavan.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+Connector.playerSelector = 'div.webPlayer';
+
+Connector.artistSelector = '.mainPanel .artist';
+
+Connector.albumSelector = '.mainPanel .album';
+
+Connector.trackSelector = '.mainPanel .song';
+
+Connector.playButtonSelector = '.playIcon';
```

Comment:
This can be dropped as you've wired up the `Connector.isPlaying` function. Alternatively and perhaps better is what @alexesprit suggested and to drop `Connector.isPlaying and wire up the `stopButtonSelector` After that  It's all good